xml2json/lib.py

```diff
@@ -19,27 +22,27 @@ def xml2json(xml_string):
 
 def convert_xml_to_json(xml, last_xmlns=None):
     try:
-        tag = unicode(xml.tag.split('}')[1])
+        tag = six.text_type(xml.tag.split('}')[1])
```

Comment:
If `six.text_type(a_string)` (without specifying an encoding) does not throw an exception in Python 3, then it wasn't necessary (`a_string` was already Unicode). Are we confident that the test coverage here is good enough that errors like that would be triggered on Python 3?

Reply:
do you mind providing an example?  I'm not sure I follow

Reply:
Sorry, I was wrong about it throwing an exception, and that's unfortunate. I still don't think this will do what you want.

```py
# Python 3.5.1
>>> import six
>>> xml_tag = b'bytes'  # imagine this is xml.tag returning a byte string
>>> six.text_type(xml_tag)
"b'bytes'"
```

I think you want something like this:
```
# next line will throw an error in Python 3 if xml.tag is bytes
# TypeError: a bytes-like object is required, not 'str'
tag = xml.tag.split('}')[1]

if six.PY2 and isinstance(tag, bytes):
    # would be better to use the encoding of the xml doc
    # leaving that as an exercise for the reader for now...
    tag = six.text_type(tag)